Makefile

```diff
@@ -10,4 +13,10 @@ fmt: FORCE
 test: FORCE
 	cairo-test -p .
 
-FORCE:
\ No newline at end of file
+language-server: FORCE
```

Comment:
I think we won't need any `Makefile` changes since we can assume after Step 3 everything is installed anyways. Instead I think we can add a `pbcopy` statement to README that I mention below.

Reply:
Got it, I'll update the pr to use `pbcopy` instead 👍